std_msgs/Header header

uint32 casualty_id

dtc_msgs/NavSatFixTimeAgo location

dtc_msgs/BoolTimeAgo severe_hemorrhage
dtc_msgs/BoolTimeAgo respiratory_distress

dtc_msgs/RateTimeAgo heart_rate
dtc_msgs/RateTimeAgo respiratory_rate

dtc_msgs/Uint8TimeAgo trauma_head
dtc_msgs/Uint8TimeAgo trauma_torso
dtc_msgs/Uint8TimeAgo trauma_upper_extremity
dtc_msgs/Uint8TimeAgo trauma_lower_extremity

dtc_msgs/Uint8TimeAgo alertness_ocular
dtc_msgs/Uint8TimeAgo alertness_verbal
dtc_msgs/Uint8TimeAgo alertness_motor
